action.yml

```diff
@@ -23,8 +23,17 @@ inputs:
   gcr_google_project:
     description: "Google project for GCR image ie:broad-jade-dev"
     default: "broad-jade-dev"
+  always_auth:
```

Comment:
This shouldn't be needed, with the old code base. It typically exports or cats out the var to a file each run sources.

Reply:
I added this b/c of the wonkiness I was seeing where stuff wasn't getting carried over between steps. I'm going to remove this before merging.